modules/sb-install-service-catalog.adoc

```diff
@@ -10,47 +10,38 @@ enable it using the procedure below.
 
 .Procedure
 
-. Create the service catalog API server.
-.. Create a file named `svcat-api-server.yaml` with the following contents.
+. Enable the service catalog API server.
+.. Use the following command to edit the service catalog API server resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogAPIServer
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogapiservers.operator.openshift.io cluster
 ----
-.. Use the following command to create the service catalog API server.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-api-server.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog API server is now installed into the `kube-service-catalog`
-namespace.
+The service catalog API server is now enabled in the
+`openshift-service-catalog-apiserver` namespace.
 
-. Create the service catalog controller manager.
-.. Create a file named `svcat-controller-manager.yaml` with the following
-contents.
+. Enable the service catalog controller manager.
+.. Use the following command to edit the service catalog controller manager resource.
 +
-[source,yaml]
 ----
-apiVersion: operator.openshift.io/v1
-kind: ServiceCatalogControllerManager
-metadata:
-  name: cluster
-spec:
-  logLevel: "Normal"
-  managementState: Managed
+$ oc edit servicecatalogcontrollermanagers.operator.openshift.io cluster
 ----
-.. Use the following command to create the service catalog controller manager.
+.. Under `spec`, set the `managementState` field to `Managed`, as shown below:
 +
 ----
-$ oc create -f svcat-controller-manager.yaml
+spec:
+  logLevel: Normal
+  managementState: Managed
 ----
+.. Save the file to apply the changes.
 +
-The service catalog controller manager is now installed into the
-`kube-service-catalog-controller-manager` namespace.
+The service catalog controller manager is now enabled in the
+`openshift-service-catalog-controller-manager` namespace.
```

Comment:
follow the wording you used in api server, but this is the right namespace you want to use for the controller-manager.